.jshintrc

```diff
@@ -28,5 +28,5 @@
   "white": false,
   "eqnull": true,
   "esnext": true,
-  "unused": true
+  "unused": false
```

Comment:
I switched this setting. I think being able to see unused arguments can still be very helpful when someone else is code spelunking. Let me know if anyone disagrees, not married to this.